src/helper/toastify.ts

```diff
@@ -1,18 +1,34 @@
 import "react-toastify/dist/ReactToastify.css";
 
-import { toast } from "react-toastify";
-import type { TypeOptions } from "react-toastify";
+import { toast } from "sonner";
+
+// import { toast } from "react-toastify";
+// import type { TypeOptions } from "react-toastify";
+
+type ToastTypes = 'success' | 'info' | 'warning' | 'error';
+
 
 /**
  * @work create a toastify by parsing the following values
  * @param {String} text
  * @param {TypeOptions} statusType
- * @param {boolean} loadingStatus
  * @returns
  */
-export function toastify(text: string, statusType: TypeOptions, loadingStatus: boolean = false) {
-  return toast(text, {
-    type: statusType,
-    isLoading: loadingStatus,
-  });
+
+
+
+export function toastify(text: string, statusType: ToastTypes) {
```

Comment:
please add the warning option as well and try whether switch case is possible

Reply:
okay on it